### ISSUE TITLE
[A11Y] File field in the registration form is not accessible

### ISSUE THREAD

**Issue by foxbunny** [indico/indico#6058]
**Describe the bug**

The field name is not mentioned when the file upload button is focused.

**To Reproduce**
1. Go to a registration form that includes a file field using a screen reader
2. Tab through the fields and focus the upload button 
3. Observe that the field name is not announced

**Expected behavior**

The field name should be included in the announcement.

**Additional context**

https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html
